internal/views/authors/index.html

```diff
@@ -18,20 +18,46 @@ <h3>Autores Cadastrados</h3>
         <tbody>
         {{range .Authors}}
             <tr>
-                <td>{{.ID}}</td>
+                <td class="idAuthor">{{.ID}}</td>
```

Comment:
```suggestion
                <td>{{.ID}}</td>
```

Essa classe não parece estar sendo usado para nada.